tests/Features/ReturnTypes/Facades.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Features\ReturnTypes;
+
+use Illuminate\Http\Client\PendingRequest;
+use Illuminate\Support\Facades\Http;
```

Comment:
Could it be that this class was not-existent in Laravel 6?

Reply:
Http got introduced in Laravel 7.